js/private/lifecycle/lifecycle-hooks.js

```diff
@@ -31,27 +31,40 @@ async function main(argv) {
         packageJson.scripts?._rules_js_postinstall
     ) {
         // node_modules folder that the node_package dependency targets which are in the same bazel package output
-        const nodeModulesPath = path.resolve(
-            path.join(outputDir, '..', '_lc', 'node_modules')
-        )
-
-        // If the package we're running postinstall on has no deps, then node_modules
-        // won't exist in the sandbox. The lifecycle functions provided by pnpm require
-        // it to exist, so create an empty folder if needed.
-        await mkdirp(nodeModulesPath)
+        const nodeModulesPath = path.resolve(path.join(outputDir, '..'))
 
+        // export interface RunLifecycleHookOptions {
+        //     args?: string[];
+        //     depPath: string;
+        //     extraBinPaths?: string[];
+        //     extraEnv?: Record<string, string>;
+        //     initCwd?: string;
+        //     optional?: boolean;
+        //     pkgRoot: string;
+        //     rawConfig: object;
+        //     rootModulesDir: string;
+        //     scriptShell?: string;
+        //     silent?: boolean;
+        //     scriptsPrependNodePath?: boolean | 'warn-only';
+        //     shellEmulator?: boolean;
+        //     stdio?: string;
+        //     unsafePerm: boolean;
+        // }
         const opts = {
             pkgRoot: path.resolve(outputDir),
             rawConfig: {
                 stdio: 'inherit',
             },
+            silent: false,
+            stdio: 'inherit',
```

Comment:
Was the `rawConfig: { stdio }` above a mistake? Can it be removed?

Reply:
Seems both are needed. Not sure what the difference is but to get the stdout & stderr out of the hook you need both 🤷